app/src/test/java/com/concordia/mcga/activities/SplashActivityTest.java

```diff
@@ -0,0 +1,18 @@
+package com.concordia.mcga.activities;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+
+@RunWith(JUnit4.class)
+public class SplashActivityTest {
+    @Test
+    public void splashScreenCreatedTest() {
+        SplashActivity splashActivity = new SplashActivity();
+        Assert.assertNotNull(splashActivity);
```

Comment:
This test is not really a test. Maybe having an Espresso test showing that it is the first Activity that starts up as soon as the app is launched. That would most probably be enough.